flit/install.py

```diff
@@ -390,7 +390,7 @@ def write_dist_info(self, site_pkgs):
             )
         self.installed_files.append(dist_info / 'direct_url.json')
 
-        with (dist_info / 'RECORD').open('w', encoding='utf-8') as f:
+        with (dist_info / 'RECORD').open('w', encoding='utf-8', newline='') as f:
```

Comment:
```suggestion
        # newline='' because the csv module does its own newline translation
        with (dist_info / 'RECORD').open('w', encoding='utf-8', newline='') as f:
```